integrations/pinterest-tag/lib/index.js

```diff
@@ -63,15 +63,27 @@ Pinterest.prototype.identify = function(identify) {
 
 Pinterest.prototype.page = function(page) {
   // If we have a category, the use ViewCategory. Otherwise, use a normal PageVisit.
+  var pinterestPageProps = {
```

Comment:
you can mutate const object properties
```suggestion
  const pinterestPageProps = {
```